### DISCUSSION TITLE
How to change the carousel arrows colors?

### DISCUSSION THREAD

**Discussion by merlinarer** [zauberzeug/nicegui#2571] [Q&A] [answered]
### Question

```py
with ui.carousel(animated=True, arrows=True, navigation=True).props('height="100%"'):
```
In the chrome styles, q-carousel__control{color:#fff} determines the arrows color, but how to know the exact pros/style parameters to set in nicegui?

**Comment by falkoschindler** [comment, marked as answer]
Hi @merlinarer, you can use the "control-color" prop:

```py
with ui.carousel(animated=True, arrows=True, navigation=True).props(r'height=100% control-color=blue'):
    with ui.carousel_slide(), ui.card().classes('bg-blue-100 w-64 h-32'):
        ui.label('Slide 1')
    with ui.carousel_slide(), ui.card().classes('bg-blue-100 w-64 h-32'):
        ui.label('Slide 2')
```

![Screenshot 2024-02-19 at 17 14 11](https://github.com/zauberzeug/nicegui/assets/5767091/0a50ea14-29f9-42fa-b298-b7ca9870c4d4)
